x-pack/plugins/triggers_actions_ui/public/application/sections/alert_details/components/alert_details.tsx

```diff
@@ -275,6 +279,49 @@ export const AlertDetails: React.FunctionComponent<AlertDetailsProps> = ({
                 </EuiFlexGroup>
               </EuiFlexItem>
             </EuiFlexGroup>
+            {!dissmissAlertErrors &&
+            alert.executionStatus.status === AlertExecutionStatusValues[2] ? (
+              <EuiFlexGroup>
+                <EuiFlexItem>
+                  <EuiCallOut
+                    color="danger"
+                    data-test-subj="alertErrorBanner"
+                    size="s"
+                    title={
+                      <FormattedMessage
+                        id="xpack.triggersActionsUI.sections.alertDetails.alertInstances.attentionBannerTitle"
+                        defaultMessage="This alert has an error caused by the {errorReason} reason."
+                        values={{
+                          errorReason: alert.executionStatus.error?.reason,
+                        }}
+                      />
+                    }
+                    iconType="alert"
+                  >
+                    <EuiTitle size="m">
+                      <h3>
+                        <EuiTextColor color="danger">
+                          <FormattedMessage
+                            id="xpack.triggersActionsUI.sections.alertDetails.alertInstances.alertErrorMessageTitle"
+                            defaultMessage="Error message:"
+                          />
+                        </EuiTextColor>
+                      </h3>
+                    </EuiTitle>
+                    <EuiText size="s" color="danger" data-test-subj="alertErrorMessageText">
+                      {alert.executionStatus.error?.message}
+                    </EuiText>
+                    <EuiSpacer size="s" />
+                    <EuiButton color="danger" onClick={() => setDissmissAlertErrors(true)}>
+                      <FormattedMessage
+                        id="xpack.triggersActionsUI.sections.alertDetails.alertInstances.dismissButtonTitle"
+                        defaultMessage="Dismiss"
```

Comment:
@mdefazio did we decide to keep or remove the dismiss button? When dismissing, the banner returns next time I refresh the page.

Reply:
If it's coming back after each refresh, then I don't think we include the dismiss option.  It would be more annoying to dismiss it and it keep coming back as opposed to always seeing and not being able to do anything with it (until the errors are fixed)